sources/packages/api/src-sql/Applications/Create-Application.sql

```diff
@@ -1,23 +1,32 @@
 CREATE TABLE IF NOT EXISTS applications (
   id SERIAL PRIMARY KEY,
   data jsonb NOT NULL,
-
   -- Reference Columns
   student_id INT REFERENCES students(id) ON DELETE CASCADE,
-
   -- Audit columns
   created_at timestamp without time zone NOT NULL DEFAULT now(),
   updated_at timestamp without time zone NOT NULL DEFAULT now(),
-  creator INT NULL DEFAULT NULL REFERENCES users(id) ON DELETE SET NULL,
-  modifier INT NULL DEFAULT NULL REFERENCES users(id) ON DELETE SET NULL
+  creator INT NULL DEFAULT NULL REFERENCES users(id) ON DELETE
```

Comment:
Is there any modification on this file? If not please remove it from the PR.

Reply:
mm, no, i guess, its just formatting